tests/queueing_honey_badger.rs

```diff
@@ -0,0 +1,150 @@
+//! Network tests for Queueing Honey Badger.
+
+extern crate hbbft;
+#[macro_use]
+extern crate log;
+extern crate env_logger;
+extern crate pairing;
+extern crate rand;
+#[macro_use]
+extern crate serde_derive;
+
+mod network;
+
+use std::cmp;
+use std::collections::BTreeMap;
+use std::iter::once;
+use std::rc::Rc;
+
+use rand::Rng;
+
+use hbbft::messaging::NetworkInfo;
+use hbbft::queueing_honey_badger::{Change, ChangeState, Input, QueueingHoneyBadger};
+
```

Comment:
Blank line in imports.

Reply:
I removed line 20 instead: I guess from the point of view of the tests, `hbbft` is an "external crate"?